### ISSUE TITLE
cannot find oid '1.3.6.1.4.1.14179.2.1.1.1.38' to walk"

### ISSUE THREAD

**Issue by eburghar** [prometheus/snmp_exporter#831]
Apparently some recent changes from `$(CISCO_URL)/AIRESPACE-WIRELESS-MIB.my` triggers the bug.

Replacing with  `http://www.circitor.fr/Mibs/Mib/A/AIRESPACE-WIRELESS-MIB.mib` fixes the error.

### Host operating system: output of `uname -a`
alpine 3.17

### snmp_exporter version: output of `snmp_exporter -version`
0.21.0

### What did you do that produced an error?
MIBDIRS=mib generator generate

### What did you expect to see?
```
ts=2023-01-02T19:48:31.659Z caller=main.go:91 level=info msg="Config written" file=/home/packager/snmp_exporter/main/snmp_exporter/src/snmp_exporter-0.21.0/generator/snmp.yml
```

### What did you see instead?
```
level=error msg="Error generating config netsnmp" err="cannot find oid '1.3.6.1.4.1.14179.2.1.1.1.38' to walk"
```

**Comment by SuperQ** [comment]
Yes, it looks like Cisco broke the MIB with a bunch of syntax errors. I filed an upstream PR to fix them.

For now, I've rolled back to a commit before it was broken. https://github.com/prometheus/snmp_exporter/pull/843
